packages/nuxt/package.json

```diff
@@ -36,6 +36,7 @@
     "@nuxt/module-builder": "^0.5.5",
     "@nuxt/schema": "^3.11.2",
     "@nuxt/test-utils": "^3.12.0",
+    "@nuxtjs/tailwindcss": "^6.12.2",
```

Comment:
not sure why i need to have it as a dev dep here but the playground fails to resolve wihtout it

Reply:
probably because the playground isn't in your pnpm workspace - maybe worth adding it?

Reply:
thanks!